app/javascript/packs/pages/Profile/Garage/index.jsx

```diff
@@ -1,22 +1,22 @@
 import React, { useState, useEffect } from "react";
 import BikeFormModal from "./BikeFormModal";
-import { Button } from "react-bootstrap";
+import { Button, Card } from "react-bootstrap";
 import { Container, Row, Col } from "reactstrap";
-import BikeShow  from './BikeShow'
+import BikeShow from "./BikeShow";
 
 const Garage = () => {
   const [modal, setModal] = useState(false);
   const toggle = () => setModal(!modal);
   const [bikes, setBikes] = useState([]);
-  const [bike, setBike] = useState({})
+  const [bike, setBike] = useState(null);
 
   const fetchMyBikes = () => {
     fetch("/api/bikes")
```

Comment:
cela renvoie un "Failed to load resource: the server responded with a status of 401 (Unauthorized)"

Reply:
Je pense que l'Unauthorized est normale, en revanche il faut faire une redirection vers le login du coup

Reply:
Yes j'ai vus que tu avais des methodses et tous avec auth mais jalors je voulais voire avec toi pour paramétrer ca.

Reply:
Ok je vais faire les signup et login bientôt, je rajoute dans mon ticket de penser à `redirect` et je ferme ici 👍